src/js/App/Header/ContextSwitcher.js

```diff
@@ -12,6 +13,15 @@ const ContextSwitcher = ({ user }) => {
   const onSelect = () => {
     dispatch(onToggleContextSwitcher());
   };
+  const dropdownItems = [
+    <DropdownItem key="link" description="PERSONAL ACCOUNT">
+      {user.identity.account_number}
+    </DropdownItem>,
+    <DropdownItem key="link">678909</DropdownItem>,
```

Comment:
The `key` prop should be unique to each children. I think we should change it to be the acc number.